### PR TITLE
Additional fields with UL lookups and SubtitleSequence class

### PR DESCRIPTION
**Pull request by daviddeelo**

### CODE REVIEW

**Comment by palemieux** [comment]
@daviddeelo I suggest using https://registry.smpte-ra.org/view/draft/Labels.xml to map ULs to symbols

File: src/main/python/cplinfo/cli.py [hunk 1]
```diff
@@ -181,15 +253,18 @@ def __init__(self, cpl_element: et.Element) -> None:
...
         edit_rate = cpl_rational_to_fraction(resource.findtext(".//cpl:EditRate", namespaces=ns_dict)) or self.edit_rate
 
         entry_point = edit_rate * int(resource.findtext(".//cpl:EntryPoint", namespaces=ns_dict) or 0)
 
-        duration = edit_rate * int(resource.findtext(".//cpl:SourceDuration", namespaces=ns_dict) or resource.findtext(".//cpl:IntrinsicDuration", namespaces=ns_dict))
+        duration = int(resource.findtext(".//cpl:SourceDuration", namespaces=ns_dict) or resource.findtext(".//cpl:IntrinsicDuration", namespaces=ns_dict)) / float(edit_rate)
 
         if duration == 0:
           continue
+        total_duration = total_duration + duration
```
Comment (palemieux) [inline src/main/python/cplinfo/cli.py]:
```suggestion
        total_duration += duration
```

File: src/main/python/cplinfo/cli.py [hunk 2]
```diff
@@ -216,6 +291,10 @@ def main():
 
   cpl_doc = et.parse(args.cpl_file)
 
+  global smpte_ul
```
Comment (palemieux) [inline src/main/python/cplinfo/cli.py]: Move to a separate source file, e.g. `ul.py`

File: src/main/python/cplinfo/smpte_ul.json [hunk 3]
```diff
@@ -0,0 +1,555 @@
+{
```
Comment (palemieux) [inline src/main/python/cplinfo/smpte_ul.json]: Use `https://registry.smpte-ra.org/view/draft/Labels.xml` instead

File: src/main/python/cplinfo/cli.py [hunk 4]
```diff
@@ -44,9 +45,18 @@ def split_qname(qname: str):
 def cpl_rational_to_fraction(r: str) -> Fraction:
   return Fraction(*map(int, r.split()))
 
+def smpte_ul_lookup(u):
```
Comment (palemieux) [inline src/main/python/cplinfo/cli.py:48]: Move to separate file python file

**Review by palemieux — CHANGES_REQUESTED** [review]

File: src/main/python/cplinfo/cli.py (hunk 4 above)
Comment (daviddeelo) [reply src/main/python/cplinfo/cli.py:48]: I moved this to a separate function.

**Pull request closed without merging**